lib/google_sign_in.rb

```diff
@@ -4,6 +4,16 @@
 module GoogleSignIn
   mattr_accessor :client_id
   mattr_accessor :client_secret
+
+  # https://tools.ietf.org/html/rfc6749#section-4.1.2.1
+  OAUTH2_ERRORS = %w[
+    invalid_request
+    invalid_client
+    invalid_grant
+    unauthorized_client
+    unsupported_grant_type
+    invalid_scope
+  ]
```

Comment:
Could stash these away in e.g. `lib/google_sign_in/errors.rb`. Exposing so apps can also validate `params[:error].presence_in(GoogleSignIn::OAUTH2_ERRORS)`